apps/todoistnext/todoist_next.star

```diff
@@ -0,0 +1,141 @@
+"""
+Applet: Todoist Next
+Summary: Todoist next due/overdue
+Description: Displays the next due or overdue task from todoist.
+Author: alisdair(https://discuss.tidbyt.com/t/todoist-integration/502/5), Updated by: akeslo
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("time.star", "time")
+load("schema.star", "schema")
+
+TODOIST_API_BASE_URL = "https://api.todoist.com/rest/v1"
+
+TODOIST_API_TASKS_URL = TODOIST_API_BASE_URL + "/tasks?filter=(overdue|today)"
+
+MODEL_KEY_TEXT = "text"
+MODEL_KEY_DUE = "due"
+MODEL_KEY_ZEN = False
+
+CACHE_KEY_MODEL = "todoist_model"
```

Comment:
Was the intention to add caching somewhere? Not sure if it's needed, but probably would be appreciated by the API provider.

Reply:
see below